### DISCUSSION TITLE
Mqtt alert is send to quickly ?

### DISCUSSION THREAD

**Discussion by mabed-fr** [blakeblackshear/frigate#11779] [General]
Hello,

I use the project frigate-notify from 0x2142

And i have very often "no snapshot available.

Read this
https://github.com/0x2142/frigate-notify/issues/80#issuecomment-2151032870

Do you think this comes from frigate's behavior with mqtt or frigate-notify?

**Comment by pseudoacacia** [comment]
[NickM-27](https://github.com/NickM-27) seems to have covered this in the thread you linked to. I just wanted to chime to say that I've got my own script that listens to frigate's MQTT messages and does various reactions for me and there's no problem with the messages coming too soon. 

If you want an image but the conditions haven't been met for a snapshot, it's easy to just pull a thumbnail and use that. Or if your snapshots are configured exactly how you want your alerts to be, then probably just wait for the update message that has a snapshot before sending a notification (because otherwise you're gong to get messages about things you didn't actually want, again assuming your snapshots are configured the way you want).
